### PR TITLE
Issue #11446: Updated testUnmappableCharacters in CheckerTest

### PR DESCRIPTION
**Pull request by SteLeo1602**
Issue: #11446

Finishing #14821 which updated verify method in testUnmappableCharacters()

### CODE REVIEW

**Comment by SteLeo1602** [comment]
CI says there are 238 characters in lines although there should be only 100. Will try to debug it later.

File: src/test/resources/com/puppycrawl/tools/checkstyle/checker/InputCheckerTestCharset.java [hunk 1]
```diff
@@ -1,3 +1,12 @@
+/*xml
+<module name="Checker">
+  <module name="com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck">
+    <property name="max" value="80"/>
```
Comment (romani) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checker/InputCheckerTestCharset.java:4]: It use to be 75

File: src/test/resources/com/puppycrawl/tools/checkstyle/checker/InputCheckerTestCharset.java [hunk 2]
```diff
@@ -1,3 +1,12 @@
...
+  </module>
+  <property name="charset" value="IBM1098"/>
+</module>
+*/
+
 package com.puppycrawl.tools.checkstyle.checker;
 
 public class InputCheckerTestCharset {
```
Comment (romani) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checker/InputCheckerTestCharset.java:12]: Please use violation below format

File: config/suppressions.xml [hunk 3]
```diff
@@ -15,6 +15,8 @@
              files="TokenTypes.java|IndentationCheckTest.java|XdocsPagesTest|CheckerTest.java"
              lines="1"/>
 
+  <suppress checks="LineLength" files="CheckerTest.java"/>
```
Comment (romani) [inline config/suppressions.xml:18]: Please remove it, let's find way to wray long lines

**Review by romani — CHANGES_REQUESTED** [review]
Item

File: src/test/resources/com/puppycrawl/tools/checkstyle/checker/InputCheckerTestCharset.java (hunk 1 above)
Comment (SteLeo1602) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checker/InputCheckerTestCharset.java]: Done

File: src/test/resources/com/puppycrawl/tools/checkstyle/checker/InputCheckerTestCharset.java (hunk 2 above)
Comment (SteLeo1602) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checker/InputCheckerTestCharset.java:12]: Done

File: src/test/resources/com/puppycrawl/tools/checkstyle/checker/InputCheckerTestCharset.java (hunk 1 above)
Comment (SteLeo1602) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checker/InputCheckerTestCharset.java]: The check reacts on config line which is longer than 75. What should I do in this situation? Can I move part of the text on the next line as I do with non-config content?

File: src/test/resources/com/puppycrawl/tools/checkstyle/checker/InputCheckerTestCharset.java [hunk 4]
```diff
@@ -1,5 +1,15 @@
+/*xml
+<module name="Checker">
+  <module name="com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck">
+    <property name="max" value="75"/>
+  </module>
+  <property name="charset" value="IBM1098"/>
+</module>
+*/
+
```
Comment (romani) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checker/InputCheckerTestCharset.java:9]: Put on this line, line 9 ,  `// violation 6 lines above`.

**Review by romani — CHANGES_REQUESTED** [review]
Remove `@SuppressForbidden` above method. This whole reason we have this update.

Item

File: src/test/resources/com/puppycrawl/tools/checkstyle/checker/InputCheckerTestCharset.java (hunk 1 above)
Comment (romani) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checker/InputCheckerTestCharset.java]: See below

File: src/test/resources/com/puppycrawl/tools/checkstyle/checker/InputCheckerTestCharset.java (hunk 4 above)
Comment (SteLeo1602) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checker/InputCheckerTestCharset.java]: Done

**Comment by romani** [comment]
No need to push to reproduce failure on local
https://dev.azure.com/romanivanovjr/romanivanovjr/_build/results?buildId=22302&view=logs&j=c902ebb4-c9f8-5f09-4e17-ff78fbbc842e&t=9ca98c81-ff64-58f0-9d03-a23ac1c4a111

Just run in terminal `mvn clean verify `.
You can do push if you want to share a problem that you can not resolve.

**Review by romani — APPROVED** [review]
Ok to merge.

This morning all CI were green, we just changed import to not violate on config comment

**Pull request merged**